kudos-board-ui/src/components/NavBar/NavBar.jsx

```diff
@@ -3,8 +3,12 @@ import SearchBar from "../SearchBar/SearchBar"
 import { useState } from "react";
 import FilterDropDown from "../FilterDropDown/FilterDropDown";
 import UISwitch from "../UISwitch/UISwitch";
+
+// Use the category table to populate the filter dropdown
 const NavBar = ({setBoards}) =>{
-    const VITE_URL = import.meta.env.VITE_URL;  
+    const VITE_URL = import.meta.env.VITE_URL;
+    // Rename this to query perhaps
+    // You could move these to the app and filter there so that way you don't have to make api calls to sort + search
     const [boardTitle, setBoardTitle] = useState("");
```

Comment:
We could move these filters to the top of the application in `App` and then you don't have to use setBoards to re-set what's being fetched and have to re-fetch to reset the boards. If you have questions about what this could look like lmk and we can go over it in person.